[ADD] Recruitment: add new job position

larm-odoo opened the pull request:
Adding a new doc for Recruitment- did not exist before. This doc explains how to add a new job position an edit the settings for the recruitment process.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/4350).

hojo-odoo commented on content/applications/hr/recruitment/new_job.rst:
```diff
@@ -0,0 +1,220 @@
+- :guilabel:`Department`: Select the relevant department for the job position.
+- :guilabel:`Expected New Employees`: Enter the number of employees to be hired for this position.
+- :guilabel:`Contract Template`: Select a contract template that will be used when offering the job
+  to the candidate.
```
```suggestion
  to a candidate.
```

hojo-odoo commented on content/applications/hr/recruitment/new_job.rst:
```diff
@@ -0,0 +1,220 @@
+Create interview form
+---------------------
+
+Once a job position has been made, the *Interview Form* needs to be created. Click on the three dots
```
Are we returning to the Kanban view to do this? If so, we should state that.

hojo-odoo commented on content/applications/hr/recruitment/new_job.rst:
```diff
@@ -0,0 +1,220 @@
+
+- :guilabel:`Layout`: Choose how the questions should be displayed.
+
+  - *One page with all the questions*: Display all sections and questions at the same time.
+  - *One page per section*: Display each section with the corresponding questions on an individual
+    page.
+  - *One page per question*: Display a single question on each page.
```
Are all of these options selectable? If so, I would use a GUI label for each.

hojo-odoo commented on content/applications/hr/recruitment/new_job.rst:
```diff
@@ -0,0 +1,220 @@
+candidate needs to pass the exam (example, 80.00%). The entry should be written in an “XX.XX”
+format.
+
+.. example::
+   A software company, *Awesome Data*, is looking for new coders to hire. Since the skill set they
+   need is very specific, an easy way to filter applicants is to give them a coding test. Scoring
+   for a coding test is a perfect solution for *Awesome Data*, so they made a test with a required
+   80% success rate. This has greatly helped their HR department save a lot of time finding the
+   right people!
```
I don't think this is necessary. It feels a bit too salesy.

hojo-odoo commented on content/applications/hr/recruitment/new_job.rst:
```diff
@@ -0,0 +1,220 @@
+
+- :guilabel:`Reward quick answers`: If the exam is to be taken live, check this box to award more
+  points to participants who answer quickly.
+- :guilabel:`Session Code`: Enter a code for participants to enter, to allow them into the
+  live exam session.
```
```suggestion
- :guilabel:`Session Code`: Enter an access code that will allow viewers into the live exam session.
```

hojo-odoo reviewed (changes requested):
Hey @larm-odoo,

Great job on this! I noticed a few instances of double spaces between words. I'm not sure if I caught them all, so that's something to keep an eye out for.

Thanks,
hojo

jcs-odoo reviewed (commented):
Hi @larm-odoo ! In general, you shouldn't remove the review requests for the teams @odoo/user-docs-structure-review and odoo/codeowner-fallback 

Doc structure (I'm part of this review team) is notified when there is a modification of the structure, such as here because you're modifying the hr.rst file, to make sure everything is in order. It's also notified when there is a change in the redirection rules (for now) so we can update the doc links in the apps.

Codeowner fallback is notified when there is no code owner defined for the modified files. This is normal in this case since you're adding a new section. I'm not on that team, but I already took care of creating the new review team, and it should be added as code owner of the new recruitment section very soon. So the codeowner coverage test should be good soon :)

About your commit message and PR title, we don't use "new" as tag, but rather [ADD]. As explained at the point 8 of [the environment setup](https://www.odoo.com/documentation/master/contributing/documentation.html#environment-setup), you can define the file [commit_template.txt](https://github.com/odoo/documentation/blob/16.0/commit_template.txt) as default commit message, which should help you write good commit messages :)

Now about your PR's content, I haven't read it, but I see the filename is `new_job.rst`. I suggest not using "new" in your filename as the doc is probably about job positions in general (including maintenance and editing) and not only about new ones (I suppose that the flow of creating a new one would rather be in a tutorial video). 

Also, if this is the main content for the Recruitment app, maybe it's best to move it to `recruitment.rst`

I hope this helps, have a good day!

larm-odoo commented:
Thanks @jcs-odoo for your explanation. I didn't add those review groups automatically to my docs, so I thought I had accidentally tagged them, which is why I removed them when I saw them listed as reviewers- I won't do that again! 

My mistake with ADD vs NEW in the PR name, but as for the file itself, it is about creating a new job, that's why the file name had new in it. I have not made the rest of the recruitment docs yet- I am doing the more details/single issue docs, then I will be making an 'overview; that will link to all the other docs. I am taking that approach to all the new HR categories, since the overviews I started to do seemed to not make a lot of sense without the supporting documentation, so I switched the order I was adding the docs.

But moving forward, I won't remove an automatically added reviewers- and we'll see if the Codeowner automatically gets added to my next PR!

larm-odoo commented on content/applications/hr/recruitment/new_job.rst:
```diff
@@ -0,0 +1,220 @@
+Job positions
+=============
+
+In the default *Recruitment* dashboard view, all job positions are shown, regardless of status.
```
From our meeting we had on Guilabels, I don't think this would get one. The name dashboard isn't truly "official", I was calling it the dashboard since that just made sense- and you don't click on anything really, you are clicking on the Recruitment app tile from the main Odoo page, and you are taken here.

StraubCreative replied:
Think in this case we're just saying the app name in a general sense, so italics is fine.

If were're suggesting (or hinting) to navigate here, we can also do it like this below. This is what I personally do because I think there's an implication to navigate here:
```suggestion
In the default :menuselection:`Recruitment` dashboard view, all job positions are shown, regardless of status.
```
If `Recruitment` is the name of the dashboard (the text heading is on the UI), a gui works too, however I don't think that's as useful, as say, calling out specific button link text during an active instruction.

All three seem to work, they just have different connotations. Up to you @larm-odoo 😉

larm-odoo replied:
OK, all things being equal, I went with menuselection @StraubCreative, since you do click on it, in some fashion at some point, so I went with that. All good now!

StraubCreative commented on content/applications/hr/recruitment/new_job.rst:
```diff
@@ -0,0 +1,210 @@
+  session.
+
+After all fields have been entered, click the :guilabel:`Save` button to save the changes, or click
+:guilabel:`Discard` to delete the changes.
```
moving this down here
```suggestion
:guilabel:`Discard` to delete the changes.

.. image:: new_job/options.png
   :align: center
   :alt: Various options to configure for the interview form.
```

StraubCreative commented on content/applications/hr/recruitment/new_job.rst:
```diff
@@ -0,0 +1,210 @@
+    individual page.
+  - :guilabel:`One page per question`: Display a single question on each page.
+
+- :guilabel:`Progression Mode`: Display the progress the candidate is making, either as a percent
+  (*percentage*), or as a number (*number*).
```
```suggestion
- :guilabel:`Progression Mode`: Display the progress the candidate is making, either as a
  :guilabel:`Percentage`, or as a :guilabel:`Number`.
```

StraubCreative commented on content/applications/hr/recruitment/new_job.rst:
```diff
@@ -0,0 +1,210 @@
+- :guilabel:`Survey Time Limit`: Check this box to limit the time allowed to complete the form. When
+  selected, a field to enter the minutes appears next to the checked box. Enter the time (using an
+  XX:XX minute/second format) in the field.
+- :guilabel:`Selection`: Display the entire form (*all questions*), or only a random selection of
+  questions from each section (*randomized per section*). The latter is not typically selected for
+  an interview form.
```
```suggestion
- :guilabel:`Selection`: Display the entire form (e.g. :guilabel:`All questions`), or only a random
  selection of questions from each section (:guilabel:`Randomized per section`).

  .. note::
     Although it is a selectable option, the :guilabel:`Randomized per section` option is not
     typically selected for an interview form.
```

StraubCreative reviewed (changes requested):
Hi @larm-odoo 
See list of changes below, for reference. I'll be pushing them up and merging in just a moment...

StraubCreative reviewed (approved):
@robodoo r+
